Tasks/AzureRmWebAppDeploymentV3/make.json

```diff
@@ -25,13 +17,37 @@
         {
             "source": "postDeploymentScript",
             "options": "-R"
+        },
```

Comment:
What are these for?

Reply:
We need to copy some non-TypeScript files over to the _build directory. Those originally just got baked into the node_module when the common package was compiled